cmd/conveyor/worker.go

```diff
@@ -30,11 +30,23 @@ var workerFlags = []cli.Flag{
 		EnvVar: "DRY",
 	},
 	cli.StringFlag{
+		Name:   "builder",
+		Value:  "conveyor",
+		Usage:  "The builder backend to use. Can be \"codebuild\" or \"docker\"",
+		EnvVar: "BUILDER",
+	},
+	cli.StringFlag{
 		Name:   "builder.image",
 		Value:  docker.DefaultBuilderImage,
-		Usage:  "A docker image to use to perform the build.",
+		Usage:  "A docker image to use to perform the build. Only used with the Docker backend.",
```

Comment:
Might be worth including the default value in this - heck, and in other flags that have default values.  I love the `%(default)s` thing you can do with argparse in python, is there a way to do that with cli?  Maybe it already shows it, it's been a while since I've dug into this library.